src/cli.ts

```diff
@@ -164,7 +164,7 @@ const getOptions = (): Result<Options> => {
     if (!str) {
       return {ok: true, value: 'no-new'};
     }
-    if (str === 'no-new' || str === 'continue') {
+    if (str === 'no-new' || str === 'continue' || str === 'kill') {
```

Comment:
I think I like the other style better because it lets TypeScript do type-narrowing, and it doesn't require allocating an array or making a function call.